docker/jukebox.Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN usermod -aG pulse ${USER}
 # Install all Jukebox dependencies
 RUN apt-get update && apt-get install -qq -y \
     --allow-downgrades --allow-remove-essential --allow-change-held-packages \
-    gcc at wget \
+    gcc g++ at wget \
```

Comment:
`gcc` is a subset of `gcc`. So if you install the first, you don't need the second. Instead maybe we should look at `build-essentials`. I am still not certain why now out of a sudden we require this instead.

Reply:
I think, if you install g++ then gcc is automatically installed, but not the other way round.

I agree installing `build-essentials` is probably the better solution.

Reply:
It still works me on Docker on Mac with `gcc`. `build-essentials` installs even more than just the compiler. Can you try with `g++` only?

Reply:
Will do

Reply:
- I checked yesterday. g++ is sufficient